app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,48 @@
+class PagesController < ApplicationController
+    before_action :set_page, only: [:show, :edit, :update, :destroy]
+    #except: [:index, :new, :create]
+
+    def index
+        @pages = Page.all
+    end
+
+    def show
+        #set_page
+        #render plain: @page.title
+    end
+
+    def new
+        @page = Page.new
+    end
+
+    def create
+        @page = Page.new(page_params)
+        @page.save
```

Comment:
I noticed you added conditionals to the Post controller for when an objects can't be created or updated but did not add them here. Is it because that was not included in the lesson?